src/hid-usage-name-overrides.json

```
@@ -8,16 +8,16 @@
     "35": { "short": "6" },
     "36": { "short": "7" },
     "37": { "short": "8" },
-    "38": { "short": "9" },
-    "39": { "short": "0" },
-    "40": { "short": "Ret", "med": "Return" },
+    "38": { "short": "9", "Name":"Keyboard 9 and Left Parenthesis"},
```

```suggestion
    "38": { "short": "9", "name":"Keyboard 9 and Left Parenthesis" },
```
To keep the conventions

---

Yep, makes sense - i'll add it as a TODO for myself if the PR gets up - good spot!